source/routes/create/bullet.js

```diff
@@ -0,0 +1,58 @@
+const PouchDB = require('pouchdb');
```

Comment:
This is from my bullet CRUD - it should definitely not be in this story and should be removed entirely.